recipes/pyutilib/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "5.3.3" %}
+
+package:
+  name: pyutilib
+  version: {{ version }}
+
+source:
+  fn: PyUtilib-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/P/PyUtilib/PyUtilib-{{ version }}.tar.gz
+  md5: 2910a329844369d4caa7fc031d64ef80
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+  entry_points:
+    - test.pyutilib=pyutilib.dev.runtests:runPyUtilibTests
+    - lbin=pyutilib.dev.lbin:main
+    - lpython=pyutilib.dev.lpython:main
+    - pypi_downloads=pyutilib.dev.pypi_downloads:main
+    - replacecopyright=pyutilib.dev.replaceCopyright:main
+    - checkcopyright=pyutilib.dev.checkCopyright:main
+    - pyutilib_test_driver=pyutilib.autotest.driver:main
+    - dispatch_srvr=pyutilib.pyro.dispatch_srvr:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose
+    - six
+
+  run:
+    - python
+    - setuptools
+    - nose
+    - six
+
+test:
+  imports:
+    - pyutilib
+    - pyutilib.autotest
+    - pyutilib.autotest.tests
+    - pyutilib.common
+    - pyutilib.component
+    - pyutilib.component.app
+    - pyutilib.component.app.tests
+    - pyutilib.component.config
+    - pyutilib.component.config.tests
+    - pyutilib.component.core
+    - pyutilib.component.core.tests
+    - pyutilib.component.executables
+    - pyutilib.component.loader
+    - pyutilib.component.loader.tests
+    - pyutilib.dev
+    - pyutilib.enum
+    - pyutilib.enum.tests
+    - pyutilib.excel
+    - pyutilib.excel.tests
+    - pyutilib.math
+    - pyutilib.math.tests
+    - pyutilib.misc
+    - pyutilib.misc.tests
+    - pyutilib.misc.tests.import_data
+    - pyutilib.misc.tests.import_data.a
+    - pyutilib.misc.tests.import_data.b
+    - pyutilib.ply
+    - pyutilib.pyro
+    - pyutilib.services
+    - pyutilib.subprocess
+    - pyutilib.subprocess.tests
+    - pyutilib.th
+    - pyutilib.th.tests
+    - pyutilib.workflow
+    - pyutilib.workflow.tests
+
+  commands:
+    - test.pyutilib --help
+    - pyutilib_test_driver --help
+    - where lbin
+    - where lpython
```

Comment:
So these would have `# [win]` selectors after them. Make sure to have at least 2 spaces between the end of the command and the beginning of the selector. Ideally the selectors should be indented so they are inline.